scripts/migrate.ts

```
@@ -0,0 +1,27 @@
+import { config } from "dotenv";
```

ESLint

[error] Unable to resolve path to module 'dotenv'.

(import-x/no-unresolved)